sourcegraph/sourcegraph
# Packages: add RubyGems support

## Test plan

Steps to test the change locally:

- Create a "Ruby Dependencies" code host, it's OK to leave `dependencies` and `repository` empty 
- Visit the URL https://sourcegraph.test:3443/rubygems/actioncable@v7.0.0/-/blob/lib/action_cable.rb the package should get cloned automatically and the `action_cable.rb` file should load
